cli-commands/vm/post/create.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+UbiRodish.on("vm").run_on("create") do
+  options("ubi vm location/vm_name create [options] public_key", key: :vm_create) do
+    on("-6", "--ipv6-only", "do not enable IPv4")
+    on("-b", "--boot_image=image_name", "boot image (ubuntu-noble,ubuntu-jammy,debian-12,almalinux-9)")
+    on("-p", "--private_subnet_id=id", "place VM into specific private subnet")
+    on("-s", "--size=size", "server size (standard-{2,4,8,16,30,60})")
+    on("-S", "--storage_size=size", "storage size (40, 80)")
+    on("-u", "--unix_user=username", "username (default: ubi)")
```

Comment:
Should we mix `-` and `_` in parameter names? `--ipv6-only` vs `--boot_image` in vm create?

Reply:
I chose the current approach only because it was simpler to have the long options match the api parameters, as the option parser uses the long option names as the hash keys by default.  It's simple to convert `-` to `_` though.  Or we could do `--ipv6_only`.  Which do you prefer?